jest.config.ts

```diff
@@ -0,0 +1,14 @@
+import type { Config } from 'jest';
+
+const config: Config = {
```

Comment:
Super unsure about this jest configuration.

Given the file workspaces. I am not sure how to configure this correctly

Reply:
What challenges are you facing?

Reply:
I'm just worried. Not sure if there needs to be special configurations with jest to work with workspaces

Reply:
I figured it out. I didn't know how to resolve loading multiple tsconfig files. with config. We can just add it to the arry in rootDirs, modulePath, and etc